BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -1676,8 +1680,10 @@ void UpdateGuidance()
                     //    TargetPosition = targetCoMPos + targetVessel.Velocity() * Time.fixedDeltaTime;
                     //}
 
-                    //increaseTurnRate after launch
-                    float turnRateDPS = Mathf.Clamp(((TimeIndex - dropTime) / boostTime) * maxTurnRateDPS * 25f, 0, maxTurnRateDPS);
+                    // Increase turn rate gradually after launch, unless vacuum steerable in space
+                    float turnRateDPS = maxTurnRateDPS;
+                    if (!(vacuumSteerable && vessel.atmDensity < 0.05f))
```

Comment:
I think I discussed this with SI (but nothing has happened about it yet) that all the `vessel.atmDensity < 0.05f` checks ought to be replaced with `vessel.InVacuum()` for consistency and the number in the `InVacuum` extension should be adjusted to `0.05f` unless there's a reason to have a `InNearVacuum` condition with a separate limit?

What happens if `boostTime = 0`? I think `Inf` gets clamped to `maxTurnRateDPS`, but `NaN` stays as a `NaN` (if `TimeIndex - dropTime` is also 0). I suspect `turnRateDPS` should just be `maxTurnRateDPS` if `boostTime == 0`.